test/models/pending_loan_test.rb

```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.